docs/customize/administration_interface.md

```diff
@@ -0,0 +1,398 @@
+# Administration panel views
+
+## Summary
+
+The following document is a developer guide to the invenio administration panel, detailing its programmatic interface and usage.
```

Comment:
After reading a bit, I think that this doc should be split in parts. It looks like that there are parts that go under topic guides, other reference. WDYT?